lambdas/search/index.py

```diff
@@ -74,7 +77,7 @@ def lambda_handler(request):
                 },
             }
         }
-        size = 0
```

Comment:
I believe size = 0 is correct. We are only interested in the aggregations and are not fetching/returning any actual documents. See: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-body.html#_fast_check_for_any_matching_docs